extensions/email/src/main/java/org/mule/extension/email/api/EmailAttributesBuilder.java

```diff
@@ -15,7 +15,6 @@
 import static javax.mail.Message.RecipientType.CC;
 import static javax.mail.Message.RecipientType.TO;
 import static org.mule.extension.email.api.EmailContentProcessor.process;
-
 import org.mule.extension.email.api.exception.EmailException;
 
 import java.time.LocalDateTime;
```

Comment:
Take in mind that this class is going to change to EmailBuilder and must have a setter for the attachments and EmailContent that are only used when the email message body is fetched.